sql/moz-fx-data-shared-prod/apple_ads_external/ios_app_campaign_stats_v1/bigconfig.yml

```diff
@@ -5,16 +5,16 @@ tag_deployments:
       name: Growth Program
     deployments:
       - column_selectors:
-          - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.apple_ads_external_derived.ios_app_campaign_stats_v1.date
-          - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.apple_ads_external_derived.ios_app_campaign_stats_v1.campaign
-          - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.apple_ads_external_derived.ios_app_campaign_stats_v1.campaign_id
-          - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.apple_ads_external_derived.ios_app_campaign_stats_v1.campaign_country_code
-          - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.apple_ads_external_derived.ios_app_campaign_stats_v1.ad_group_name
-          - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.apple_ads_external_derived.ios_app_campaign_stats_v1.ad_group_id
+          - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.apple_ads_external.ios_app_campaign_stats_v1.date
```

Comment:
I'm not sure why the project is written twice but that looks to be correct based on all the other configs

Reply:
Yeah, that's an annoyance in Bigeye. Basically the naming is `<data source>.<project ID>.<dataset>.<table>`. The "data source" in our case is `moz-fx-data-shared-prod`, since it's set up for that project. So the duplication is correct

Reply:
Oops, my bad. Sorry about that and thanks for fixing it @BenWu 🙇